apps/hash-ai-worker-ts/src/activities/shared/get-llm-response.ts

```diff
@@ -235,21 +229,6 @@ export const getLlmResponse = async <T extends LlmParams>(
                       subjectId: aiAssistantAccountId,
                     },
                   },
-                  {
-                    relation: "setting",
-                    subject: {
-                      kind: "setting",
-                      subjectId: "viewFromWeb",
-                    },
-                  },
-                  {
-                    relation: "viewer",
-                    subject: {
-                      kind: "accountGroup",
-                      subjectId: hashInstanceAdminGroupId,
-                      subjectSet: "member",
-                    },
-                  },
```

Comment:
We were going to just comment these out, and others below, so that we can do a big check later once other Cedar permissions are implemented, that we have properly translated them (except where we decided to do something differently)